File: Arrays/Richest-Customer-Wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int mx = Integer.MIN_VALUE;
        for(int i=0; i<accounts.length; i++){
            int temp = 0;
            for(int j=0; j<accounts[i].length; j++){
                temp += accounts[i][j];
            }
            mx = Math.max(mx, temp);
        }
        return mx;
    }
}

File: Arrays/Richest-Customer-Wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:

        mx = -sys.maxsize

        for i in range(0, len(accounts)):
            temp = 0
            for j in range(0, len(accounts[i])):
                temp += accounts[i][j]
            mx = max(mx, temp)

        return mx
